tests/test_fft.py

```diff
@@ -1,62 +1,40 @@
-"""
-In these tests, we iterate through some valid fft backends in for [common.fft] attribute in config.ini
-using the `config_override` mechanism for `Config` objects, and check for valid results.
-
-Because the determination of which fft backend to use happens at import time, this is somewhat tricky to do.
-Here we employ the `importlib.reload` function while we're iterating through these fft backends to explicitly
-trigger a reload.
-"""
-
-from importlib import reload
 from unittest import TestCase
-
-import aspire.utils.numeric
-from aspire.utils.config import config_override
+import numpy as np
+from aspire.utils.numeric import fft_class
 
 fft_backends = ('scipy', 'pyfftw')
```

Comment:
So CuPy isn't tested?

Reply:
The Cupy lib is not in the default install list. But the unit tests for Cupy were added back separately in commit 49c5bf0.

Reply:
All right, but now the tests for Cupy duplicate those of SciPy/FFTW. It would be better to include it as an element in the `fft_classes` array and test it like the others. For this to work, you have switch out `np` for `xp` again as it was from the beginning to make sure that you have access to `cp` when testing the Cupy FFT.

Reply:
In the commit 3537c48, both Cupy fft class and numeric class were added to the lists for looping all these cases.